index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style1.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Playfair+Display:ital,wght@1,500&display=swap"
+        rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@200&display=swap" rel="stylesheet">
+    <title>Document</title>
+</head>
+
+<body>
+    <div class="nav-container">
+        <div class="wrapper">
+            <img class="headerimgHolder" src="assets/image_geometry_1.png" alt="headerimgHolder" />
+            <div class="ic_menu">
+                <a><img src="assets/ic_menu.png" alt="menu icon"></a>
+            </div>
+        </div>
+
+    </div>
+
+    <div class="container">
+        <div class="title_name">
+            Himesh Suthar
+        </div>
+        <div class="about_description">
+            Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my
+            works.
+            If you like what you see and have a project you need coded, don’t hesitate and contact me.
+        </div>
+        <div class="collaborate_button">
+            start collaboration
+        </div>
+
+    </div>
+    <div class="container">
+
+        <img class="image_geometry_1" src="assets/image_geometry_2-1.png" alt="image_geometry_2-1">
+
+    </div>
+
+    <div class="project_container">
+        <div class="project_title_holder">
+            Project
+        </div>
+
+        <div class="container">
+            <img class="Rectangle_image" src="assets/Rectangle-21.png" alt="Rectangle_image" />
+
+        </div>
```

Comment:
- [ ] Please make sure your design match with the original template design👏 
- Your design
![screenshot (31)](https://user-images.githubusercontent.com/44624138/143325900-10654abe-35e4-4943-9c19-56b1713d3c4c.png)
- Figma design
![screenshot (29)](https://user-images.githubusercontent.com/44624138/143325924-e2b99d79-6993-4c43-93ae-dca3f6b69e3d.png)

Reply:
![screenshot](https://user-images.githubusercontent.com/60375984/143476217-b46900ff-5eda-48f1-a858-8e39829bc258.png)
 Done